article-store/events/utils.py

```diff
@@ -0,0 +1,122 @@
+from contextlib import contextmanager
+from functools import partial, wraps
+import json
+from time import gmtime, strftime
+from typing import (
+    Any,
+    Callable,
+    ContextManager,
+    Dict
+)
+import uuid
+
+import pika
+from pika.exceptions import ConnectionClosed
+from pika.adapters.blocking_connection import BlockingChannel
+
+from .conf import (
+    ARTICLES_EXCHANGE,
+    BROKER_PARAMS,
+    DEFAULT_QUEUES
+)
+
+
+DELIVERY_MODE_PERSISTENT = 2
+
+
+def create_message(msg_type: str, identifier: str, data: Dict) -> Dict:
+    """Create a message `dict` based on a standard schema.
+
+    :return: Dict
+    """
+    return {
+        "eventId": str(uuid.uuid1()),
+        "happenedAt": strftime("%Y-%m-%dT%H:%M:%S+00:00", gmtime()),
+        "aggregate": {
+            "service": "article-store",
+            "name": "article-version",
```

Comment:
may be `article` if this is an `article.delete`-related event?

Reply:
I'd say comment out the `aggregate` field for now and wait for https://github.com/libero/walking-skeleton/pull/28/files to add it later when stable

Reply:
Ok, the article id is in `aggregate.identifier` currently. Should it be moved elsewhere for the time being?